app/src/ui/terms-and-conditions/terms-and-conditions.tsx

```diff
@@ -71,10 +71,11 @@ export class TermsAndConditions extends React.Component<
 
           <p>
             If you configure the Software to work with one or more accounts on
-            the GitHub.com website or with an instance of GitHub Enterprise,
-            your use of the Software will also be governed your applicable
-            GitHub.com website Terms of Service and/or the license agreement
-            applicable to your instance of GitHub Enterprise ("GitHub Terms").
+            the GitHub.com website or with an instance of GitHub Enterprise
```

Comment:
I'm not sure we want to change the wording in the terms and conditions. @billygriffin do you know?

Reply:
actually nvm i think this is good!